Gemfile

```diff
@@ -1,3 +1,8 @@
 source "https://rubygems.org"
 
+group :development do
```

Comment:
I think we could lose the `group :development` throughout. None of these Gemfiles are part of the released gem. They are only used in test and development.